docs/policy_cookbook/metadata_based_synchronization.md

```diff
@@ -0,0 +1,10 @@
+# Metadata-based Locking
```

Comment:
this title does not match the filename - so one or the other should probably get updated.

Reply:
Will correct that. The filename is the correct name.

Still a WIP

Reply:
Fixed title

Reply:
filename itself?